moveit_drake.repos

```diff
@@ -13,8 +13,8 @@ repositories:
     version: main
   moveit_resources:
     type: git
-    url: https://github.com/moveit/moveit_resources
-    version: ros2
+    url: https://github.com/sjahr/moveit_resources
```

Comment:
Why do you need a new branch of this?

Reply:
It has the .obj files already, so no need to run the script again

Reply:
Do you intend to merge in with this? I guess it's easier for quick testing... but eventually we may want to go back on this and link to the main upstream.

Reply:
Yes, I intend to merge this for now. Alternatively, we could make everybody applying this script as an install step :thinking: Let's merge this for now and change it as a follow-up